packages/esm-patient-registration-app/src/widgets/cancel-patient-edit.test.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import CancelPatientEdit from './cancel-patient-edit.component';
+
+describe('CancelPatientEdit component', () => {
+  const mockClose = jest.fn();
+  const mockOnConfirm = jest.fn();
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('renders the modal and triggers close and onConfirm functions', () => {
+    const { getByRole } = render(<CancelPatientEdit close={mockClose} onConfirm={mockOnConfirm} />);
+
+    const cancelButton = getByRole('button', { name: /Cancel/i });
```

Comment:
Can we do it like this?

```suggestion
    render(<CancelPatientEdit close={mockClose} onConfirm={mockOnConfirm} />);

    const cancelButton = screen.getByRole('button', { name: /Cancel/i });
```